app/locales/en.json

```diff
@@ -100,23 +100,34 @@
     "launch_set_up_phone": "Set up my phone",
     "legal_page_title": "Legal",
     "loading_public_data": "loading data...",
-    "location_disabled_message": "COVID Safe Paths requires location services.",
-    "location_disabled_title": "Location Tracking Was Disabled",
-    "location_enabled_message": "COVID Safe Paths is securely storing your GPS coordinates once every five minutes on this device.",
-    "location_enabled_title": "COVID Safe Paths Enabled",
+    "location_disabled_message": "Please enable location access for Safe Paths.",
+    "location_disabled_title": "Device location disabled",
+    "location_enabled_message": "Safe Paths is anonymously recording your location.",
+    "location_enabled_title": "Device location Enabled",
+    "location_stopped_message": "Stopped at user request",
     "logging_active": "Location Active",
     "logging_inactive": "Location Inactive",
+    "maps_import_button_text": "Import past locations",
+    "maps_import_disclaimer": "Safe Paths has no affiliation with Google and never shares your data.",
+    "maps_import_text": "To see if you encountered someone with COVID-19 prior to downloading this app, you can import your personal location history.",
+    "maps_import_title": "Google Maps",
```

Comment:
These maps strings look like they came from a bad merge.

Try running `yarn i18n:extract` to remove any unused keys.